azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -249,23 +247,13 @@ internal class CallCompositeActivity : AppCompatActivity() {
             NavigationStatus.IN_CALL -> {
                 supportActionBar?.setShowHideAnimationEnabled(false)
                 supportActionBar?.hide()
-                requestedOrientation = if (isAndroidTV(this)) {
```

Comment:
We should not remove (isAndroidTV(this))
Otherwise this behaviour will be changed for TV applications.

Reply:
Adding SCREEN_ORIENTATION_FULL_SENSOR as an API Option for all TV.